src/http/endpoints/update_profile.rs

```diff
@@ -0,0 +1,86 @@
+use crate::ayb_db::db_interfaces::AybDb;
+use crate::ayb_db::models::{Entity, InstantiatedEntity, Link};
+use crate::error::AybError;
+use crate::http::structs::EntityPath;
+use crate::http::utils::unwrap_authenticated_entity;
+use crate::http::verification::is_verified;
+use crate::http::web_frontend::WebFrontendDetails;
+use actix_web::{post, web, HttpResponse};
+use serde::Deserialize;
+use std::str::FromStr;
+use url::Url;
+
+#[derive(Deserialize)]
+struct ProfileLinkUpdate {
+    pub url: String,
+}
+
+#[derive(Deserialize)]
+struct ProfileUpdate {
+    pub display_name: Option<String>,
+    pub description: Option<String>,
+    pub workplace: Option<String>,
+    pub links: Vec<ProfileLinkUpdate>,
+}
+
+#[post("/v1/entity/{entity}")]
```

Comment:
Not that we're RESTful anyway, but just thinking through POST/PUT/PATCH

Reply:
I think PATCH is more accurate, in the previous implementation POST made more sense, but now it doesn't :p